battle/StageBasis.java

```diff
@@ -143,12 +143,7 @@ public StageBasis(BattleField bf, EStage stage, BasisLU bas, int[] ints, long se
 		canon = new Cannon(this, nyc[0], nyc[1], nyc[2]);
 		conf = ints;
 
-		if(st.minSpawn <= 0 || st.maxSpawn <= 0)
-			respawnTime = 1;
-		else if(st.minSpawn == st.maxSpawn)
-			respawnTime = st.minSpawn;
-		else
-			respawnTime = st.minSpawn + (int) ((st.maxSpawn - st.minSpawn) * r.nextDouble());
+		respawnTime = 0;

```

Comment:
initial spawn time is based on min/max respawn time set for that stage (e.g. 1~60f for earthshaker) so i'd like that segment to still be there